### ISSUE TITLE
Tier II Starlight Crafting Altar is correct but the recipes wont work 

### ISSUE THREAD

**Issue by TomKing1** [DarkPacks/SevTech-Ages#1657]
![bingo](https://user-images.githubusercontent.com/38212845/38501110-1f250122-3c04-11e8-9aa3-9795af016a1d.PNG)

**Comment by daddy77gaming** [comment]
What are you crafting there? is that a coal engine? 
It seems that recipe is missing in 3.0.6 age 2
celestial altar pre 3.0.6 had 18 recipe's now i see only 17

**Comment by TomKing1** [comment]
Yeah, it's not just that recipe that's messing up I'm gonna go back to 3.0.5

**Comment by TomKing1** [comment]
now I'm wondering if its my fault all the recipes from the previous versions work

**Comment by daddy77gaming** [comment]
just to make sure.. 

YOU CAN craft your coal engine in a Version pre 3.0.6 ?

YOU are missing the coal engine age2 recipe in 3.0.6 ?

**Comment by SummitSummit** [comment]
Ah, I see I'm not the only one having this problem.  I also can NOT craft the Starlight Infuser.

**Comment by mojax37** [comment]
I cannot create recipes on anything above the first tier altar. I am playing on a server and my teammate can use the altars just fine. We have all the same achievements. No idea what is causing this.

**Comment by SummitSummit** [comment]
I can make most things on the Celestial Altar, just not the Coal Engine and the Starlight Infuser.

**Comment by Pe4enie** [comment]
Same issue reported [here](https://github.com/Darkosto/SevTech-Ages/issues/1731)

**Comment by Mesukiel** [comment]
I'm in 3.0.6 and had the same problem of the altar bar turning red and not recognizing the structure built for it.  I tried both with a resource pack and without from a fresh install of 3.0.6.

- I rebuilt the structure in creative in the same world and it worked.  
- Went back to survival, tore down and rebuilt structure and altar did not work.  
- Then I started the project over and didn't use chisel to make the marble blocks needed for the 
  structure, only using the recipes in Astral Sorcery.  Once altar structure was rebuilt it then worked.  

The blocks needed are can be chiseled from marble but the resulting structure will not work.  Only blocks that have been created using the altar from Astral Sorcery seem to work.  Hope this helps.

Mesu
